UiPath.Ipc.net6.0-windows.received.txt

```diff
@@ -0,0 +1,221 @@
+[assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/UiPath/coreipc.git")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Playground")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.CoreIpc.BackCompat")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.CoreIpc.Extensions.Abstractions")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.CoreIpc.Tests")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.Ipc.Tests")]
+[assembly: System.Runtime.Versioning.SupportedOSPlatform("Windows7.0")]
+[assembly: System.Runtime.Versioning.TargetFramework(".NETCoreApp,Version=v6.0", FrameworkDisplayName=".NET 6.0")]
+[assembly: System.Runtime.Versioning.TargetPlatform("Windows7.0")]
+namespace UiPath.Ipc
+{
+    public readonly struct CallInfo
+    {
+        public CallInfo(bool newConnection, System.Reflection.MethodInfo method, object?[] arguments) { }
+        public object?[] Arguments { get; }
+        public System.Reflection.MethodInfo Method { get; }
+        public bool NewConnection { get; }
+    }
+    public abstract class ClientTransport : System.IEquatable<UiPath.Ipc.ClientTransport>
+    {
+        protected ClientTransport() { }
+    }
+    public class EndpointCollection : System.Collections.Generic.IEnumerable<UiPath.Ipc.EndpointSettings>, System.Collections.IEnumerable
+    {
+        public EndpointCollection() { }
+        public void Add(System.Type type) { }
+        public void Add(UiPath.Ipc.EndpointSettings endpointSettings) { }
+        public void Add(System.Type contractType, object? instance) { }
+        public System.Collections.Generic.IEnumerator<UiPath.Ipc.EndpointSettings> GetEnumerator() { }
+    }
+    public sealed class EndpointNotFoundException : System.ArgumentException
+    {
+        public string EndpointName { get; }
+        public string ServerDebugName { get; }
+    }
+    public class EndpointSettings : System.IEquatable<UiPath.Ipc.EndpointSettings>
+    {
+        public EndpointSettings(System.Type contractType, System.IServiceProvider serviceProvider) { }
+        public EndpointSettings(System.Type contractType, object? serviceInstance = null) { }
+        public System.Func<UiPath.Ipc.CallInfo, System.Threading.CancellationToken, System.Threading.Tasks.Task>? BeforeIncomingCall { get; set; }
+        public System.Type ContractType { get; }
+        public System.Threading.Tasks.TaskScheduler? Scheduler { get; set; }
+        public object? ServiceInstance { get; }
+        public System.IServiceProvider? ServiceProvider { get; }
+        public void Validate() { }
```

Comment:
I see you've added ServiceProvider and scheduler to IpcBase - so we'd be good to remove them from here, right?